packages/common/src/content/get-family.ts

```diff
@@ -0,0 +1,44 @@
+import { HubFamily } from "../types";
+import { getCollection } from "../collections";
+
+// private helper functions
+function collectionToFamily(collection: string): string {
+  const overrides: any = {
+    other: "content",
+    solution: "template",
+  };
+  return overrides[collection] || collection;
+}
+
+/**
+ * return the Hub family given an item's type
```

Comment:
Please add a little clarity about what family is and how it relates to types

Reply:
Will do in a future PR. This PR only moves this fn.